### PR TITLE
refactored wish + wishgroup

### PR DESCRIPTION
**Pull request by cameronl10**

### CODE REVIEW

File: backend/src/resolvers/wishGroup.ts [hunk 1]
```diff
@@ -1,19 +1,20 @@
-import Pool from '../../config/dbConnect';
+import { getWishGroup, editWishGroupColor, editWishGroupTitle, CreateWishGroup, DeleteWishGroup, getWishes } from '../handlers/wishGroupHandler';
```
Comment (jaykbpark) [inline backend/src/resolvers/wishGroup.ts]: I think we can just import the entire wishgroupHandler file for each resolver file

File: backend/src/resolvers/wishGroup.ts [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 
 const resolvers = {
     Query: {
-        wishGroup: async (_: any, { houseID, title, }: any): Promise<WishGroup> => {
-            return await getWishGroup(houseID, title);
+        wishGroup: async (_: any, { houseid, title, }: { houseid: String, title: String }): Promise<WishGroup> => {
```
Comment (jaykbpark) [inline backend/src/resolvers/wishGroup.ts:5]: remove console.log please

**Review by jaykbpark — CHANGES_REQUESTED** [review]
requesting changes -- please check comment and re-request 👍

**Comment by cameronl10** [comment]
@jaypark25 made changes thx pookie 😻

File: backend/src/resolvers/household.ts [hunk 3]
```diff
@@ -1,27 +1,25 @@
...
     },
     Mutation: {
-        createHousehold: async (_: any, { household }: any): Promise<Household> => {
-            return await CreateHousehold(household);
+        createHousehold: async (_: any, { household }: {household: Household}): Promise<Household> => {
+            return await householdHandler.CreateHousehold(household);
         },
-        editHousehold: async (_: any, { household }: any): Promise<void> => {
-            return await EditHousehold(household);
+        editHousehold: async (_: any, { household }: {household: Household}): Promise<void> => { // should this be a houseid instead of a whole household?
```
Comment (jaykbpark) [inline backend/src/resolvers/household.ts:14]: You mean the input? I think it should be a whole household because we need optional var inputs?

**Review by jaykbpark — APPROVED** [review]
LGTM thanks!

**Pull request merged**